pkg/webui/console/views/device-general-settings/application-server-form/index.js

```diff
@@ -163,13 +181,20 @@ const ApplicationServerForm = React.memo(props => {
     >
       {showResetNotification && <Notification content={messages.keysResetWarning} info small />}
       <Form.Field
-        autoFocus
+        required
```

Comment:
```suggestion
```
Radio button options can never be `required` since there is always an option selected already.